src/lib.rs

```diff
@@ -126,15 +126,49 @@ pub trait Renderable: Send + Sync {
     fn render(&self, context: &mut Context) -> Result<Option<String>>;
 }
 
+pub type TemplateLocation = Path;
+pub trait FileRepository {
```

Comment:
Shopify/liquid historically called included templates `snippet`s. More recently, they've added custom blocks/tags that load templates from other folders, such as [`section`](https://help.shopify.com/themes/liquid/tags/theme-tags#section).

After this PR, I'd really like liquid-rust to support non-`include` that also load templates.